.travis.yml

```diff
@@ -1,10 +1,8 @@
 language: node_js
 node_js:
-  - "0.8"
-  - "0.10"
   - "0.12"
   - "4.2"
-  - "5"
+  - "5.5"
```

Comment:
Not my intention. Changing to major version now. Will do it for version 4 as well.